test/html.test.js

```diff
@@ -980,3 +1021,39 @@ QUnit.test('audio segments with info trigger audioinfo event', function() {
   QUnit.equal(infoEvents.length, 2, 'audio info should trigger');
   QUnit.deepEqual(infoEvents[1].info, newinfo, 'audio info = muxed info');
 });
+
+QUnit.test('creates native SourceBuffers immediately if a second ' +
+  'VirtualSourceBuffer is created', function() {
```

Comment:
this line needs to be spaced over so that it matches the indent of the line above it: 
``` JavaScript
QUnit.test('fooo bar' +
                 'bar foo', function() {
```